jupyterlab_polus_render/README.md

```diff
@@ -1,119 +1,152 @@
+3. Dragging-and-dropping the dataset.
 </br>
 
-Please note that usage differs significantly from https://pypi.org/project/polus-render/0.0.4.0.1.5/
-
 <img src="images/home.png"/>
 
-# Requirements
-* Python 3.9+
+## Requirements
+
+* JupyterLab >= 4.0
+
+## Install
 
-# Installation
+You can install using `pip`:
+
+```bash
+pip install jupyterlab_polus_render
 ```
-pip install "git+https://github.com/PolusAI/jupyterlab-extensions.git#egg=jupyterlab_polus_render&subdirectory=jupyterlab_polus_render"
-export RENDER_URL="<Polus Render URL>"
+
+## Uninstall
+
+To remove the extension, execute:
+
+```bash
+pip uninstall jupyterlab_polus_render
 ```
 
-You may also run `%env RENDER_URL <Polus Render URL>` in a notebook to set the enviromental variable.
-You will need to restart Jupyter Server for `render-server-ext` endpoints to take effect.
+## Troubleshoot
 
-# Project File Structure
+If you are seeing the frontend extension, but it is not working, check
+that the server extension is enabled:
+
+```bash
+jupyter server extension list
 ```
-jupyterlab_polus_render
-| LICENSE
-| MANIFEST.in                     // Packaging entries
-| pyproject.toml                  // Pypi config 
-| README                          
-└───render-server-ext             // Server extension used by jupyterlab_polus_render
-└───polus
-    | polus_render.py             // Main file, contains render function used by user
+
+If the server extension is installed and enabled, but you are not seeing
+the frontend extension, check the frontend extension is installed:
+
+```bash
+jupyter labextension list
 ```
 
-# Build Instructions
-- cd to `jupyterlab_polus_render` root directory.
-- `py -m build`
-- `py -m twine upload  dist/*`
-- Enter `__token__` as user and reference API keys for password
+## Sample usage
+```Python
+import jupyterlab_polus_render
+
+# Initiates the static build of Render with height of iFrame window set to 900px.
+jupyterlab_polus_render.Render(height = 900)
+
+# Serves a tiff image by providing the local path. 
+jupyterlab_polus_render.Render(imagePath = 'images/LuCa-7color_3x3component_data.ome.tif', height = 900)
+
+# Serves a tiff image by providing the local path along with an overlay used for the image. 
+jupyterlab_polus_render.Render(imagePath = 'images/LuCa-7color_3x3component_data.ome.tif', overlayPath = 'images/overlay_render2.json', height = 900)
+
+# Embeds an iFrame of a static build of Polus Render with remote image
+jupyterlab_polus_render.Render(imagePath = 'https://viv-demo.storage.googleapis.com/LuCa-7color_3x3component_data.ome.tif', height = 900)
 
-## NOTE:
-- For each upload, version number must be changed in `pyproject.toml`
-- Add additional files to `MANIFEST.in` to bundle them with Pypi package
+# Serves a zarr dataset by providing the local path along with an overlay used for the dataset. 
+jupyterlab_polus_render.Render(imagePath = 'images/pyramid.zarr', overlayPath = 'images/overlay_render2.json', height = 900)
```

Comment:
- Remove `height` parameter
- Use `Render` class directly since we imported it above